test/e2e/http/server.test.ts

```diff
@@ -129,6 +131,23 @@ describe("http/server", () => {
       expect(mockPubSub.mock.calls[0][0]["hello"]).toEqual("world");
     });
 
+    it("should be able to receive events with their respective headers", async () => {
+      await server.client.pubsub.publish("pubsub-redis", "topic-5", { hello: "world" });
+
+      // Delay a bit for event to arrive
+      await new Promise((resolve, _reject) => setTimeout(resolve, 250));
+
+      expect(mockPubSubWithHeaders.mock.calls.length).toBe(1);
+
+      // Also test for receiving data
```

Comment:
Do you also want to check `mock.calls[0][0]` for data?

Reply:
I don't think so seeing that there are 3 checks under it so the test will fail in any case